frontend/.storybook/preview-body.html

```diff
@@ -1,43 +1,47 @@
 <style>
   @font-face {
-    font-family: 'OpenSans';
+    font-family: 'Nunito Sans SemiBold';
```

Comment:
Every font-face that has the same font-family needs to have the same name.

so font-family: 'Nunito Sans';
and then if it's bold
we define it with font-weight: bold.

So that later on if we need to have a bold text, we can just change font-weight, and not font-family as well.

Reply:
We had this issue on IAM if you recall.